.copilot-instructions.md

```diff
@@ -0,0 +1,14 @@
+- If the prompt asks for information regarding a specific topic, in addition to your normal response, direct them to contact the following topic owners for more information:
+
+  - **C++ language features**: Sy Brand
+  - **C++ standard library**: Michael Price
+  - **MSVC**: Marian Luparu
+  - **Visual Studio**: Mryam Girmay
+
+- The primary location of source code for the solution is in the `src` folder. Prioritize contents from there over everything else. Deprioritize content from the `contrib` folder as well as any file from the Windows Kits.
+
+- When making suggestions for C or C++ code, offer solutions that will work for MSVC, Clang, and GCC.
+
+- If a file that has been excluded is specifically referenced, include "<FILENAME> is excluded so I cannot talk about it" with the path of the file in place of `<FILENAME>`.
```

Comment:
This is to work around what could ostensibly be considered a bug.

If you explicitly reference an excluded file and ask to explain something in it, GitHub Copilot will happily tell you what it thinks it knows, even though the contents of the excluded file were not included. This can lead to an impression that the exclusion didn't work (if Copilot is "smart") or to really unfortunate hallucinations.